### PR TITLE
[Async Messages]Add assets

### PR DESCRIPTION
**Pull request by ipavlidakis**
### 🎯 Goal

Add the assets required by Async Messages

### 🛠 Implementation

Just adding resource files

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### CODE REVIEW

File: Sources/StreamChatUI/Resources/Assets.xcassets/chevron.left.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "chevron.backward.png",
```
Comment (martinmitrevski) [inline Sources/StreamChatUI/Resources/Assets.xcassets/chevron.left.imageset/Contents.json:4]: Have you tested how everything looks on versions older than iOS 13?

File: Sources/StreamChatUI/Appearance+Images.swift [hunk 2]
```diff
@@ -59,6 +59,21 @@ public extension Appearance {
         public var emptyChannelListMessageBubble: UIImage = loadImageSafely(with: "empty_channel_list_message_bubble")
         public var download: UIImage = loadSafely(systemName: "icloud.and.arrow.down", assetsFallback: "download")
 
+        // MARK: - Recording
+
+        public var mic: UIImage = loadSafely(systemName: "mic", assetsFallback: "mic")
```
Comment (martinmitrevski) [inline Sources/StreamChatUI/Appearance+Images.swift:64]: Maybe it's best if you update this page in this PR, so we don't forget it: https://getstream.io/chat/docs/sdk/ios/common-content/reference-docs/stream-chat-ui/appearance.images/.

File: Sources/StreamChatUI/Appearance+ColorPalette.swift [hunk 3]
```diff
@@ -16,6 +16,7 @@ public extension Appearance {
         /// Static color which should stay the same in dark and light mode, because it's only used as text on small UI Elements
         /// such as `ChatUnreadCountView`, `GiphyBadge` or Commands icon.
         public var staticColorText: UIColor = .streamWhiteStatic
+        public var staticBlackColorText: UIColor = .streamBlackStatic
```
Comment (martinmitrevski) [inline Sources/StreamChatUI/Appearance+ColorPalette.swift:19]: Where do we need a black static color? Does it have an impact on dark mode?

Comment (ipavlidakis) [reply Sources/StreamChatUI/Appearance+ColorPalette.swift:19]: The play/pause button, retains its white background and black icon color even on dark mode. We are using the `staticBlackColorText` for the color of the icon.

Screenshot 2023-05-09 at 2 59 26 PM

File: Sources/StreamChatUI/Resources/Assets.xcassets/chevron.left.imageset/Contents.json (hunk 1 above)
Comment (ipavlidakis) [reply Sources/StreamChatUI/Resources/Assets.xcassets/chevron.left.imageset/Contents.json:4]: I haven't 😞 but i'm downloading iOS 13 simulator as we speak

Comment (ipavlidakis) [reply Sources/StreamChatUI/Resources/Assets.xcassets/chevron.left.imageset/Contents.json:4]: Tested ✅ iPhone Xs iOS 13.7
|Dark - Recording|Dark - Locked|Light - Recording|Light - Locked|
|:-------------: |:-------------:| -----:|-----:|
|Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 54 12|Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 54 04|Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 53 52|Simulator Screen Shot - iPhone Xs - 2023-05-09 at 17 53 44|

**Pull request merged**